ambry-api/src/main/java/com/github/ambry/config/ReplicationConfig.java

```diff
@@ -181,6 +182,17 @@
   @Default("")
   public final Set<String> replicationVcrRecoveryPartitions;
 
+  /**
+   * To specify the type of replication to be used for inter colo replication.
+   * It has two values - "all-to-all" or "leader-based". Default value is "all-to-all".
```

Comment:
nit: please update comment to reflect latest change. It should be "ALL_TO_ALL" and "LEADER_BASED".